### DISCUSSION TITLE
ImportError for 'Bg'

### DISCUSSION THREAD

**Discussion by wsande** [python-cmd2/cmd2#1174] [Q&A]
I'm trying to follow the Initialization example, which includes a banner.  But I'm getting an ImportError"

`ImportError: cannot import name 'Bg' from 'cmd2'`

code: (copied from the example)
```
import cmd2
from cmd2 import (
    Bg,
    Fg,
    style,
)
```

**Comment by kmvanbrunt** [comment]
What version of cmd2 are you using? `Bg` wasn't added until 2.3.0.
